content-multi-sample/src/main/content/jcr_root/apps/commerce/demandware/config.author/com.adobe.cq.commerce.demandware.replication.transport.WebDAVTransportPlugin.config

```diff
@@ -0,0 +1,2 @@
+webdav.user="{FILL ME}"
```

Comment:
this must be either configured in the demandwareclient or the webdav plugin needs to be a factory: the two instances might have to use different credentials..